cypress/e2e/po/lists/machine-pools-list.po.ts

```diff
@@ -9,4 +11,27 @@ export default class MachinePoolsListPo extends BaseResourceList {
   downloadYamlButton() {
     return new ResourceTablePo(this.self()).downloadYamlButton();
   }
+
+  machinePoolCount(poolName: string, count: any, options?: GetOptions) {
```

Comment:
`any` type, the count parameter is typed as any. Since you are passing both numbers (`2`) and Regex (`/^2$/`) in the spec file, we should tighten this up.

```javascript
machinePoolCount(poolName: string, count: string | number | RegExp, options?: GetOptions)
```